.env.example

```diff
@@ -4,6 +4,7 @@
 #
 MYUSA_KEY: consumer_public_key
 MYUSA_SECRET: consumer_secret_key
+DEFAULT_URL_HOST: your_local_server_address
```

Comment:
I have it set to `http://localhost:5000` locally and that is working for me . http://guides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views